pkg/docker/cockpit-atomic-storage

```diff
@@ -405,15 +410,19 @@ def print_info(data):
         sys.stdout.flush()
 
 def cmd_monitor():
-    # Figure out whether we can manage the pool.  For that, we need
-    # both a volume group to work with, and a recent enough version of
-    # the "atomic" utility.
+    # Figure out whether we can manage the pool.  For that, we
+    # need a recent enough version of the "atomic" utility.
     global can_manage
     can_manage = False
     try:
-        if get_dss_vgroup() != "":
-            with open("/dev/null", "w") as dev_null:
-                can_manage = call([ "atomic", "storage", "--help" ], stdout=dev_null) == 0
+        with open("/dev/null", "w") as dev_null:
```

Comment:
I no longer see dev_null being used here. Can you help me understand what's going on?